crates/bevy_render/src/primitives/mod.rs

```diff
@@ -81,11 +93,29 @@ impl Sphere {
     }
 }
 
-/// A bisecting plane that partitions 3D space into two regions.
+/// A region of 3D space, specifically a closed set which border is a bisecting 2D plane.
+/// This bisecting plane is partitionning 3D space into two "infinite" regions,
+/// the half-space is one of those regions and includes the bisecting plane.
+///
+/// Each instance of this type is characterized by:
+/// - the bisecting plane's unit normal, normalized and pointing "inside" the half-space,
+/// - the distance from the bisecting plane to the origin of 3D space along the normal,
+///
+/// The distance can also be seen as:
+/// - the distance from the origin of 3D space to the bisecting plane along the inverse of the normal,
+/// - the opposite of the distance from the origin of 3D space to the bisecting plane along the normal.
```

Comment:
To me, it would make more sense intuitively that the distance would be from the origin to the plane along the normal (so the opposite of what it is) so I wanted to put the emphasis on that.